core/ui-common/src/main/kotlin/com/wire/android/ui/theme/WireColorPalette.kt

```diff
@@ -317,4 +317,7 @@ object WireColorPalette {
     val Pink = Color(0xFFEB239B)
     @Stable
     val Turquoise = Color(0xFF01718E)
+
+    @Stable
+    val SmokeyGray = Color(0xFF737373)
```

Comment:
This color is not a part of our [design color palette](https://www.figma.com/design/JGiz61Nl8WCobwPkHoF9yZ/Documentation?node-id=3091-15033) so it shouldn't be added to it